### PR TITLE
Move form styles from styles.css to inputs.css

### PR DESCRIPTION
**Pull request by Henni**
Let's modularize our stylesheet(s).
This PR moves all styles regarding forms (inputs, buttons, etc.) to a separate file.

@jancborchardt @owncloud/designers

### CODE REVIEW

**Comment by jancborchardt** [comment]
@Henni nice! One formal change though, I would call it »inputs.css« cause that’s a bit more understandable.

**Comment by Henni** [comment]
@jancborchardt fixed

File: core/css/inputs.css [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+input[type="email"],
+input[type="url"],
+input[type="time"] {
+	-webkit-appearance:textfield; -moz-appearance:textfield;
+	-webkit-box-sizing:content-box; -moz-box-sizing:content-box; box-sizing:content-box;
+}
+input[type="text"]:hover, input[type="text"]:focus, input[type="text"]:active,
+input[type="password"]:hover, input[type="password"]:focus, input[type="password"]:active,
+input[type="number"]:hover, input[type="number"]:focus, input[type="number"]:active,
+input[type="search"]:hover, input[type="search"]:focus, input[type="search"]:active,
```
Comment (Henni) [inline core/css/inputs.css]: short heads up: this line previously looked like the following
```css
.searchbox input[type="search"]:hover, .searchbox input[type="search"]:focus, .searchbox input[type="search"]:active
```

Comment (jancborchardt) [inline core/css/inputs.css]: Cool – this is the only real change except for moving and whitespace change?

**Comment by MorrisJobke** [comment]
I tested this and it still works :+1:

Also the changes look good

File: core/css/inputs.css (hunk 1 above)
Comment (Henni) [inline core/css/inputs.css]: Yes it is.

**Comment by jancborchardt** [comment]
Awesome work @Henni! :+1: 

I also just removed my old outdated »inputs« branch where I started to do the same way back when. ;D

Merge plz @DeepDiver1975

**Pull request merged**